frontend/packages/dev-console/integration-tests/features/pipelines/pipelines-Runs.feature

```diff
@@ -1,18 +1,17 @@
 Feature: Pipeline Runs
-    As a user I want to start pipeline, rerun, delete pipeline run
+    As a user, I want to start pipeline, rerun, delete pipeline run
 
 Background:
-    Given user logged into the openshift application
-    And openshift cluster is installed with pipeline operator
-    And user is at the project namespace "AUT_MB_Demo" in dev perspecitve
+    Given user has installed OpenShift Pipelines operator
+    And user has selected namespace "aut-pipelines-runs"
+    And user is at pipelines page
 
 
 @regression
-Scenario Outline: Start pipeline popup details for pipeline with one resource : P-04-TC02
-    Given user is at the Pipelines page
-    And pipeline "<pipeline_name>" consists of task "<task_name>" with one git resource
-    When user selects "start" from the kebab menu
-    Then "Start Pipeline" popup displays with Git Resources, Advanced Options sections
+Scenario Outline: Start pipeline popup details for pipeline with one resource : P-04-TC02    
```

Comment:
Can you please remove outline from Scenario Outline from every scenario?

Reply:
Scenario Outline is mandatory when you are using examples

Reply:
Yes. But never use Scenario Outline. We just use Scenario